src/main/java/Homework1/n_1_specialized_not_done.java

```diff
@@ -0,0 +1,143 @@
+package Homework1;
+
+import java.util.Scanner;
+
+public class n_1_specialized_not_done {
+
+    public static void main(String[] args) {
+
+        Scanner in = new Scanner(System.in);
+
+        System.out.println("Сколько лет вашей собаке сейчас?");
+
+        int x = in.nextInt();
+
+        System.out.println("Сколько ему(ей) месяцев?");
+
+        int z = in.nextInt();
+
+        System.out.println("Сколько лет вы хотите прибавить?");
+
+        int y = in.nextInt();
+
+        System.out.println("Сколько месяцев вы хотите ему прибавить?");
+
+        int f = in.nextInt();
+
+        if (x + y < 5)
+
+            System.out.println("Так у вас же ещё щенок! Ему и 5 лет даже нету – ему всего лишь 3 годика.");
+
+        else {
+
+            if (z == 0 & f == 0) {
+
+                if ((x + y) % 10 >= 5 || (x + y) % 10 == 0 || (x + y) % 100 == 11) {
```

Comment:
I can't understand, what does 10 mean? what does 100 mean?
Also please give a more meaningful names to yours variables, it is very difficult to read... e.g. int x ->int dogYears